File: python/scripts/user_schema.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
user_schema = travelperk.scim().discovery().user_schema()
write_output("user_schema", [
    user_schema["id"],
    user_schema["name"],
    user_schema["description"],
    user_schema["attributes"],
    user_schema["meta"]["resourceType"],
    user_schema["meta"]["location"],
])

File: php/scripts/user_schema.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$userSchema = $travelperk->scim()->discovery()->userSchema();
write_output("user_schema", [
	$userSchema->id,
	$userSchema->name,
	$userSchema->description,
	$userSchema->attributes,
	$userSchema->meta->resourceType,
	$userSchema->meta->location,
]);
